README.md

```diff
@@ -1,5 +1,12 @@
 # Specification 'Web of Things (WoT) Thing Description'
 
+<div align="center">
+  
+[![Follow on Twitter](https://img.shields.io/twitter/follow/W3C_WoT.svg?label=follow+W3C_WoT)](https://twitter.com/W3C_WoT)
+<img alt="Stack Exchange questions" src="https://img.shields.io/stackexchange/stackoverflow/t/web-of-things?style=plastic">
```

Comment:
Very cool addition to the readme :) I was wondering, however, if the badge should also link to Stackoverflow (for example https://stackoverflow.com/questions/tagged/web-of-things). If I am not mistaken, clicking on the badge just opens the SVG in another tab.

Reply:
@JKRhb you are right, a surrounding `<a>` is needed to work as expected.

Reply:
Using Markdown syntax I guess this could work?

```suggestion
[![Stack Exchange questions](https://img.shields.io/stackexchange/stackoverflow/t/web-of-things?style=plastic)]( https://stackoverflow.com/questions/tagged/web-of-things)
```